client/pk-monitor.c

```diff
@@ -296,6 +341,28 @@ main (int argc, char *argv[])
 		goto out;
 	}
 
+	/* use the bus to resolve connetcion names to PIDs */
+	bus_conn = g_bus_get_sync (G_BUS_TYPE_SYSTEM, NULL, &error);
+	if (bus_conn == NULL) {
+		g_printerr ("Cannot connect to the system bus: %s", error->message);
```

Comment:
```suggestion
		g_printerr ("Cannot connect to the system bus: %s\n", error->message);
```

`g_printerr()` doesn’t automatically add a trailing newline.